apps/console/src/public/deployment.config.json

```diff
@@ -29,7 +29,7 @@
     },
     "idpConfigs": {
         "enablePKCE": true,
-        "clockTolerance": 60,
+        "clockTolerance": 300,
```

Comment:
Since this file is templated, If this value should be the default, make sure this change done to the [default.json](https://github.com/brionmario/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json) or configured as default on the SDK side.

If this change is needed only for a specific env, the corresponding toml config can be found [here](https://github.com/brionmario/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json).

Reply:
Looks like in default json this is not added yet. So let's merge this PR and update the default.json in the next PR. @thivi Please update after.